bait-files/src/main/webapp/submission.js

```diff
@@ -0,0 +1,9 @@
+/** Do not allow spaces in input */
+function checkSpace(event)
+{
+   if(event.which ==32)
+   {
+      event.preventDefault();
+      return false;
+   }
+}
```

Comment:
Return true at the end of this function.

It's kind of weird to see a function that possibly returns false down one code path (the if statement) but doesn't return at all in another code path.  I'm actually not sure what gets returned otherwise... maybe "undefined" haha.. but I'm not sure.